app/assets/images/payment_icons/sezzle.svg

```diff
@@ -1 +1 @@
+<svg width="38" height="24" viewBox="0 0 38 24" version="1.1" xmlns="http://www.w3.org/2000/svg" role="img" aria-labelledby="pi-sezzle"><title id="pi-sezzle">Sezzle</title><defs><linearGradient x1="80.524%" y1="85.62%" x2="45.753%" y2="37.68%" id="pi-sezzle-linearGradient-1"><stop stop-color="#CE5DCB" offset="0%"/><stop stop-color="#C558CC" offset="20.95%"/><stop stop-color="#AC4ACF" offset="55.25%"/><stop stop-color="#8534D4" offset="98.45%"/><stop stop-color="#8333D4" offset="100%"/></linearGradient><linearGradient x1="-.001%" y1="49.995%" x2="99.997%" y2="49.995%" id="pi-sezzle-linearGradient-2"><stop stop-color="#FF5667" offset="2.37%"/><stop stop-color="#FC8B82" offset="65.92%"/><stop stop-color="#FBA28E" offset="100%"/></linearGradient><linearGradient x1="-.003%" y1="50%" x2="100%" y2="50%" id="pi-sezzle-linearGradient-3"><stop stop-color="#00B874" offset="0%"/><stop stop-color="#29D3A2" offset="51.26%"/><stop stop-color="#53DFB6" offset="68.17%"/><stop stop-color="#9FF4D9" offset="100%"/></linearGradient><linearGradient x1="-.003%" y1="49.999%" x2="99.999%" y2="49.999%" id="pi-sezzle-linearGradient-4"><stop stop-color="#FCD77E" offset="0%"/><stop stop-color="#FEA500" offset="52.41%"/><stop stop-color="#FF5B00" offset="100%"/></linearGradient></defs><g transform="translate(1 1)" fill-rule="nonzero" stroke="none" stroke-width="1" fill="none"><rect stroke-opacity=".07" stroke="#000" stroke-width=".96" fill="#FFF" x="-.48" y="-.48" width="36.96" height="22.96" rx="1.92"/><path d="M1.36 13.482a4.643 4.643 0 006.566 0l.174-.174c.906-.906-.907-5.66 0-6.566l-6.74 6.74z" fill="url(#pi-sezzle-linearGradient-1)" transform="translate(10 1)"/><path d="M8.275 6.567l-.174.174c-.906.906.906 5.66 0 6.566l6.74-6.74a4.63 4.63 0 00-3.283-1.36 4.624 4.624 0 00-3.283 1.36" fill="url(#pi-sezzle-linearGradient-2)" transform="translate(10 1)"/><path d="M1.36 6.916a4.643 4.643 0 000 6.566l6.916-6.916a4.643 4.643 0 000-6.566L1.36 6.916z" fill="url(#pi-sezzle-linearGradient-3)" transform="translate(10 1)"/><path d="M7.926 13.482a4.643 4.643 0 000 6.566l6.916-6.916a4.643 4.643 0 000-6.566l-6.916 6.916z" fill="url(#pi-sezzle-linearGradient-4)" transform="translate(10 1)"/></g></svg>
```

Comment:
The border radius is slightly different than other icons. Is it possible to adjust it?
[Video demonstration](https://screenshot.click/09-26-u0am9-yw3aq.mp4)

Thank you

Reply:
@tauthomas01 hopefully it's fixed now, thank you!